src/main/test/at/ac/fhcampuswien/fhmdb/MovieSearchTest.java

```diff
@@ -0,0 +1,149 @@
+package at.ac.fhcampuswien.fhmdb;
+
+import at.ac.fhcampuswien.fhmdb.features.MovieSearchService;
+import at.ac.fhcampuswien.fhmdb.models.Movie;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class MovieSearchTest {
+
+    private final List<Movie> movies = Movie.initializeMovies();
```

Comment:
Create some movies here (hardcoded), because Movie.initializeMovies(); is dependent on the working initializeMovies Method. So the unit test wouldn't be independent anymore.

Reply:
Resolved